android_openssl.cmake

```diff
@@ -1,26 +1,30 @@
 function(add_android_openssl_libraries)
   if(CMAKE_BUILD_TYPE STREQUAL "Debug")
-    set(ssl_root_path ${CMAKE_CURRENT_FUNCTION_LIST_DIR}/no-asm)
+    set(SSL_ROOT_PATH ${CMAKE_CURRENT_FUNCTION_LIST_DIR}/no-asm)
   else()
-    set(ssl_root_path ${CMAKE_CURRENT_FUNCTION_LIST_DIR})
+    set(SSL_ROOT_PATH ${CMAKE_CURRENT_FUNCTION_LIST_DIR})
   endif()
 
   if(Qt6_VERSION VERSION_GREATER_EQUAL 6.5.0)
-    if(NOT OPENSSL_ROOT_DIR)
-      set(OPENSSL_ROOT_DIR ${SSL_ROOT_PATH}/ssl_3/${CMAKE_ANDROID_ARCH_ABI})
-    endif()
     list(APPEND android_extra_libs
```

Comment:
we no longer need to assign this android_extra_libs list if we're doing it for OPENSSL_SSL_LIBRARY and OPENSSL_INCLUDE_DIR, we can use those latter two variables directly for QT_ANDROID_EXTRA_LIBS